### PR TITLE
QDM::PhysicalExamOrder, QDM::PhysicalExamPerformed, QDM::PhysicalExamRecommended

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/cqm_report.rb [hunk 1]
```diff
@@ -52,6 +52,8 @@
 require_relative 'qrda-import/data-element-importers/medication_discharge_importer.rb'
 require_relative 'qrda-import/data-element-importers/medication_dispensed_importer.rb'
 require_relative 'qrda-import/data-element-importers/patient_characteristic_expired.rb'
+require_relative 'qrda-import/data-element-importers/physical_exam_order_importer.rb'
```
Comment (mayerm94) [inline lib/cqm_report.rb]: suggest moving the physical_exam_performed line to be with these

File: lib/qrda-import/patient_importer.rb [hunk 2]
```diff
@@ -39,6 +39,8 @@ def initialize
         @data_element_importers << generate_importer(MedicationDischargeImporter)
         @data_element_importers << generate_importer(MedicationDispensedImporter)
         @data_element_importers << generate_importer(PatientCharacteristicExpired)
+        @data_element_importers << generate_importer(PhysicalExamOrderImporter)
```
Comment (mayerm94) [inline lib/qrda-import/patient_importer.rb]: same comment as above, suggest moving physicalexamperformedimporter to be with these
